snappi_ixnetwork/ixnetworkapi.py

```diff
@@ -285,6 +285,29 @@ def add_error(self, error):
         else:
             self._errors.append(error)
 
+    def parse_location_info(self, location):
+        """It will return (chassis,card,port)
+        set card as 0 where that is not applicable"""
+        if re.search("/|;", location) is None:
+            raise Exception("Please specify location with supported connection status")
```

Comment:
exception message is not clear, as the condition is search for "/|;", message shall be some thing like
"Invalid port location format, expected [chassis_ip;card_id;port_id or chassis_ip/port_id(UHD only)]"